solana/README.md

```diff
@@ -103,5 +106,42 @@ In order to run them, the Solana programs must be built and their IDL made avail
 To ensure the SDK has the generated IDL, run the tests with the make command:
 
 ```sh
-make anchor-test
-```
\ No newline at end of file
+make test
+```
+
+### Troubleshooting
+
+<details>
+<summary><code>make: *** No rule to make target `test'.  Stop.</code></summary>
+
+- Ensure `Makefile` has target `test`
+</details>
+
+<details>
+<summary><code>tsx: command not found</code></summary>
+
+- Screenshot:
+  <img src="images/tsx-command-not-found.png" alt="tsx command not found screenshot">
+- Update `Makefile` ([line #29](https://github.com/wormhole-foundation/example-native-token-transfers/blob/main/solana/Makefile#L29)) from:
+
+  ```sh
+  tsx scripts/regenerateIdl.ts $$jsonfile > $$tsfile; \
+  ```
+
+  to:
+
+  ```sh
+  npx tsx scripts/regenerateIdl.ts $$jsonfile > $$tsfile; \
+  ```
+
+  </details>
+
+  <details>
+  <summary><code>Lifecycle script `build:esm` failed with error</code></summary>
+
+  - Screenshot:
+    <img src="images/lifecycle-script.png" alt="lifecycle script screenshot">
+  - This occurs due to Typescript files failing compilation.
+  - [`patch-idl` script](https://github.com/wormhole-foundation/example-native-token-transfers/blob/main/solana/scripts/patch-idl) requires [`jq`](https://jqlang.github.io/jq/) to be installed. Install `jq` and retry.
```

Comment:
again, maybe the prerequisites section would be a better place for this.

also we should make the error explicitly warn about this